src/declarations/stencil-public-compiler.ts

```diff
@@ -1255,10 +1259,30 @@ export interface ResolveModuleIdResults {
   pkgDirPath: string;
 }
 
+/**
+ * A controller which provides for communication and coordination between
+ * threaded workers.
+ */
 export interface WorkerMainController {
+  /**
+   * Send a given set of arguments to a worker
+   */
   send(...args: any[]): Promise<any>;
-  handler(name: string): (...args: any[]) => Promise<any>;
+  /**
+   * Handle a particular method
+   *
+   * @param name of the method to be passed to a worker
+   * @returns a Promise wrapping the results
+   */
+  handler(name: WorkerContextMethod): (...args: any[]) => Promise<any>;
+  /**
+   * description
+   *
+   */
```

Comment:
Whoops - looks like a placeholder made its way in here

Reply:
whoops, thought I got them all! 😅